src/main/common/filter.h

```diff
@@ -70,9 +70,9 @@ typedef enum {
 } filterType_e;
 
 typedef enum {
-    KD_FILTER_CLASSIC = 0,
```

Comment:
Hate to do this to ya PaPaYou but I think we should revert this change and keep them as KD_FILTER_XXXX - just so the code clarity is improved.

SP / CLASSIC / NOSP are fine for people familiar with the genesis of these features but for anyone else that comes in on this code it might be confusing. The KD_FILTER add some clarity by making it immeditaly obvious that its dealing with D term filters.

Reply:
Yes sure no problem